xpt2046/xpt2046.go

```diff
@@ -0,0 +1,175 @@
+// Package xpt2046 implements a driver for the XPT2046 resistive touch controller as packaged on the TFT_320QVT board
+//
+// Datasheet: http://grobotronics.com/images/datasheets/xpt2046-datasheed.pdf
+package xpt2046
+
+import (
+	"machine"
+	"time"
+
+	"tinygo.org/x/drivers/touch"
+)
+
+type Device struct {
+	t_clk  machine.Pin
+	t_cs   machine.Pin
+	t_din  machine.Pin
+	t_dout machine.Pin
+	t_irq  machine.Pin
+
+	precision uint8
+}
+
+func New(t_clk, t_cs, t_din, t_dout, t_irq machine.Pin, precision uint8) Device {
```

Comment:
I think it would be better to remove the precision setting from New.
It would be better to use Configure(config *Config) instead, where precision can be set.

Reply:
Done